guide.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Fun Yo-yo Facts</title>
+		<h1>Fun Facts on Yo-yos!</h1>
+</head>
+<body>
+	<fieldset>
+		<table border=2>
+			<tr>
+				<th>Brand</th>
+				<th>Model</th>
+				<th>Type</th>
+			</tr>
+			<tr>
+				<td>CLYW</td>
+				<td><a href="Images/summit.jpg">Summit</a></td>

```

Comment:
these are anchor tags.  if you want to link an image to a page you should look into the `<img>` tag